docs/GettingStarted.md

```diff
@@ -1 +1,25 @@
-# Getting Started
\ No newline at end of file
+# Getting Started
+
+## Solution contents
+The Power-Platform-Local-Gov-Component-Library.zip solution contains:
+- **Master - LocalGov Components** - The component library containing the Address, Email, TelephoneNumber and Date components.
+- **Example App** - A typical canvas application which has all the components on 2 separate user journeys. This can be used as guidance when creating custom functionality and theming for your own applications.
+- **LocalGov Components Glossary** - A canvas app displaying each component for exploration of the library and component functionality.
+
+## Importing the solution into an environment
+1. From Power Apps, go to Solutions tab in Power Apps Studio.
+2. Click Import solution and choose the zip file you downloaded then click Import. The import may take some time, be patient.
+
+## Importing the components into an application
+The components themselves are contained in the Master - LocalGov Components component library. To use the components in your own canvas application, follow these steps:
+1. Open a canvas application in the environment you downloaded and imported the solution in.
+2. Go to Insert > Get more components (file and magnifying glass symbol) > Master - LocalGov Components > Select all components needed > Import.
```

Comment:
If I'm using things correctly, there's 2 Insert menus, one at the top and one on the left, and only the one on the left can be used to import? might be worth specifying that?